### PR TITLE
Release next

### PR DESCRIPTION
**Pull request by scsmithr**
# Features

Add pages/update existing pages. Mention in Jan 2024 release.

- [x] Clickhouse (@scsmithr)
- [x] BSON table funcs (@tychoish)
- [x] ~hashing table funcs (`partition_results`, etc) (@tychoish)~
- [x] `read_sqlserver` (@scsmithr)
- [x] COPY TO bson (@tychoish)
- [x] #209 (@greyscaled)
- [x] Cloud SQL Workspace (@greyscaled)
- [x] Cloud Connection Dialog (@greyscaled)
- [x] Cloud Auth Github (@greyscaled)
- [x] ~Cassandra~ https://github.com/GlareDB/glaredb.github.io/issues/218

# Fixes

Just mention in Jan 2024 release.

- [x] Fix iceberg table version reading

### CODE REVIEW

**Comment by greyscaled** [comment]
To double check: https://github.com/GlareDB/glaredb/compare/v0.7.1...main

**Comment by tychoish** [comment]
there are PRs open on the meta repo with blog posts about all of the things with my name next to them.

File: index.md [hunk 1]
```diff
@@ -104,9 +104,14 @@ allocation. GlareDB Cloud is available to everyone with a
 
 See our [Pricing page] for more details on available plans.
 
+[S3]: /docs/data-sources/supported/s3/
```
Comment (tychoish) [inline index.md:107]: do these links get rendered in the source using the _actualy_ domain and path or do we rely on redirection (and does that matter for SEO?)

Comment (scsmithr) [reply index.md]: These get rendered in the source. And we have a sitemap so seo should be fine (https://docs.glaredb.com/sitemap.xml)

**Comment by scsmithr** [comment]
> We haven't documented any scalar functions which the hashing and partitioning functions count as, and it feels weird to put them in the same category as the table functions.

Crossed it out. We'll want to add those in at some point, and maybe generate some/most of the documentation from glare catalog. Could be interesting to just generate a cheat sheet for scalars ordered by name or something.

**Review by greyscaled — APPROVED** [review]

**Pull request merged**